src/components/debate/Rebuttal.vue

```diff
@@ -1,254 +1,402 @@
+        </div>
+        <!-- Add comment form -->
+        <div class="add-comment-form w-100" v-show="addComment">
+          <div class="create-comment-form w-100 p-3 mb-2">
+            <div class="w-100 row m-0 justify-content-center p-3">
+              <h3
+                class="
+                  create-post-close
+                  rounded-circle
+                  row
+                  m-0
+                  align-items-center
+                  justify-content-center
+                "
+                v-on:click="toggleComment"
+              >
+                x
+              </h3>
+            </div>
+            <Create></Create>
+          </div>
+        </div>
+      </div>
+      <!-- ......List component..... -->
     </div>
+  </div>
 </template>
 
-
 <script>
+import Create from "@/components/debate/Create.vue";
+import Comment from "@/components/debate/Comment.vue";
 
-import For from "@/components/debate/For.vue"
-import Against from "@/components/debate/Against.vue"
-import Create from "@/components/debate/Create.vue"
+import { rebuttalService } from "@/services";
+import { config } from "@/configurations";
+import { getSanitizedTime } from "@/helpers";
 export default {
-    name: 'List',
-    components: {For, Against, Create},
-    data(){
-        return{
-            addComment:false,
-            rebuttal: false,
-            fFor: true,
-            fAgainst: true
-        }
+  name: "Rebuttal",
+  components: { Create, Comment },
+  props: ["rebuttalItem", "rebuttalTime", "uuid"],
+  watch: {
+    uuid: function (newVal, oldVal) {
+      
+      this.rebuttals = null;
+      console.log("Prop changed: ", newVal, " | was: ", oldVal);
+      this.loadRebuttals();
     },
-    methods:{
-        toggleComment(){
-            if(this.addComment){
-                this.addComment = false
-            } else {
-                this.addComment = true
-            }
-        },
-
-        toggleRebuttals(){
-             if(this.rebuttal){
-                this.rebuttal = false
-            } else {
-                this.rebuttal = true
-            }
-        },
-
-        filterAll(){
-            this.fFor = true
-            this.fAgainst = true
-        },
-        filterFor(){
-            this.fFor = true
-            this.fAgainst = false
-        },
-        filterAgainst(){
-            this.fFor = false
-            this.fAgainst = true
-        }
-    }
-}
+  },
+  created() {
+    this.loadRebuttals();
+  },
+  data() {
+    return {
+      rebuttals: null,
+      addComment: false,
+      openRebuttal: false,
+      inclination: null,
+    };
+  },
+  computed: {
+    comment: function () {
+      if (this.rebuttalItem) {
+        return this.rebuttalItem.comment;
+      } else {
+        return "";
+      }
+    },
+    profile: function () {
+      if (this.rebuttalItem) {
+        return this.rebuttalItem.profile.user;
+      } else {
+        return "";
+      }
+    },
+    profilePic: function () {
```

Comment:
use `v-if` directly inside the image tag instead of creating a function. This applies to all such occurrences for this PR.